packages/lodestar/src/api/impl/beacon/state/utils.ts

```diff
@@ -92,3 +104,13 @@ export function getEpochBeaconCommittees(
   }
   return committees;
 }
+
+function stateToApiStateContext(config: IBeaconConfig, state: BeaconState | null): ApiStateContext | null {
```

Comment:
I don't think we ever need epochCtx for archive state. That why ApiStateContext allows epochCtx to be null. So I don't think we need that method and unfinelized state comes with it anyways